src/sass/pages/home.scss

```diff
@@ -43,6 +43,7 @@
       border: 0;
       white-space: normal;
       font-size:18px;
+      -webkit-appearance:none;
```

Comment:
I added it because our create account button was getting no padding on
iPhone and this made it have the padding style that we're giving it.
On 27 Aug 2015 13:40, "Raunak Kathuria" <notifications@github.com> wrote:

> In src/sass/pages/home.scss
> <https://github.com/binary-com/binary-static/pull/405#discussion_r38063782>
> :
>
> > @@ -43,6 +43,7 @@
> >        border: 0;
> >        white-space: normal;
> >        font-size:18px;
> > +      -webkit-appearance:none;
>
> we don't need these prefix, we have grunt module to append it and why only
> webkit?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/binary-com/binary-static/pull/405/files#r38063782>.
>